docs/MigrationGuide.md

```diff
@@ -0,0 +1,58 @@
+# Migration guide from POMI (nri-prometheus) to newrelic-prometheus
+
+What have changed:
+POMI (nri-prometheus) was a custom solution build by New Relic with some of the Prometheus Server capabilities. The `newrelic-prometheus` solution is actually running the Prometheus Server in Agent mode. This allows to use any scrape related feature of the Prometheus Server like authorization methods and relabel configs.
+Also we have built a `prometheus-configurator` on top of that which improved the configuration experience of the solution.
+
+## Metadata
+
+Found details [here](./MetricLabels.md) about the labels added by `newrelic-prometheus`.
+
+This is a list of POMI metadata and it replacement in the new solution.
+
+| POMI | newrelic-prometheus | Note |
+|------|---------------------|------|
+| `namespaceName`| `namespace` | Renamed |
+| `nodeName`| `node` | Renamed |
+| `podName`| `pod` | Renamed |
+| `serviceName`| `service` | Renamed |
+| `deploymentName`| - | Removed |
+| `clusterName` | `cluster_name` | Renamed |
+| `integrationName` | - | Removed |
+| `integrationVersion` | - | Removed |
+| `metricName` | - | Removed |
+| `nrMetricType` | - | Removed |
+| `promMetricType` | - | Removed |
+| `label.<kubernetesLabel>`| `<kubernetesLabel>` | Renamed - The `label.` prefix has been removed and the label name sanitized to Prometheus naming standards. |
+| `targetName` | - | Removed |
+| `scrapedTargetKind` | `job` | Replaced - The `job` can be used to identify the scrape kind |
+| `scrapedTargetName` | - | Removed |
+| `scrapedTargetURL` | `instance` | Replaced - `instance` contains the `host:port` for the target |
+
+## Kubernetes target discovery
+
+The target discovery configuration has improved with the introduction of the Jobs and can be easily configured as explained [here](./KuberntesTargetFilter.md).
+
+There are some default behaviors that differs on the solutions.
+By default POMI scrapes Pods and Services that has `prometheus.io/scrape=true` Label or Annotation. But `newrelic-prometheus` scrapes Pods and Endpoints from Services with `prometheus.io/scrape=true` Annotation. 
```

Comment:
Do you think it is worth mentioning POMI here? We assume all the users know what is POMI 🤔

Reply:
I have made the reference on top and also in the title of the doc. have you check that? not sure if it the best approach to mention all the names every time is referred.